spinalcordtoolbox/reports/assets/index.html

```diff
@@ -100,4 +112,70 @@
 <script src="_assets/js/bootstrap-table.min.js"></script>
 <script src="_assets/js/main.js"></script>
 <script>var sct_data = $sct_json_data;</script>
-</html>
+<script>
+    function toggleColumn(buttonID){
+        /*
+        This function will hide a specific column of the table if it is displayed
+        or show a specific column if it is not displayed.
+        */
+        var position;
+        position = findPosition(buttonID);
+        if (document.getElementById(buttonID).innerHTML == "Hide " + buttonID){
+                document.getElementById(buttonID).innerHTML = "Hide " + buttonID + " \u2714";
+                document.getElementsByTagName('TH')[position].style.display = "none";
+                var iTD;
+                for (iTD = position; iTD < document.getElementsByTagName('TD').length; iTD = iTD + 4){
+                    document.getElementsByTagName('TD')[iTD].style.display = "none";
+                }
+        }
+        else if (document.getElementById(buttonID).innerHTML == "Hide " + buttonID + " \u2714"){
+                document.getElementById(buttonID).innerHTML = "Hide " + buttonID;
+                document.getElementsByTagName('TH')[position].style.display = "table-cell";
+                var iTD;
+                for (iTD = position; iTD < document.getElementsByTagName('TD').length; iTD = iTD + 4){
+                    document.getElementsByTagName('TD')[iTD].style.display = "table-cell";
+                }
+        }
+    }
+
+    function findPosition(buttonID){
+        /*
+        This function will find the array position
+        of the header cell corresponding to the wanted Id.
+        Return type: number
+        */
+        var iTH, position;
+        for (iTH = 0; iTH < document.getElementsByTagName('TH').length; iTH++){
+            if (buttonID.valueOf() == document.getElementsByTagName('TH')[iTH].innerText){
+                position = iTH;
+            }
+        }
+        return position;
+     }
+
+    let body = document.body;
+    let dropdown = document.querySelector('.dropdown');
+    let btn = document.querySelector('.btn');
+
+    function manualOpenClose(e){
+        /*
+        This function toggles the opening and closing of
+        the dropdown-menu when the button for the dropdown is clicked.
+        */
+        e.stopPropagation();
+        dropdown.classList.toggle('open');
+    }
+
+    function stopDropdownClose(e){
+        /*
+        This function keeps the inside contents of the dropdown-menu
+        open even if the inside contents or the body of the html file are clicked.
+        */
+        e.stopPropagation();
+    }
+
+    body.onclick = stopDropdownClose;
+    btn.onclick = manualOpenClose;
+
+</script>
+</html>
```

Comment:
add carriage return

Reply:
Ok! Added in commit [156df39](https://github.com/neuropoly/spinalcordtoolbox/commit/156df39d228c7d127a52204e1ff903b3b43a1750)